src/Cdn/test/Invoke-AzCdnCommitProfileMigration.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Invoke-AzCdnCommitProfileMigration'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Invoke-AzCdnCommitProfileMigration.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Invoke-AzCdnCommitProfileMigration' {
+    It 'Commit' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
```

Comment:
Please write test cases for cmdlets

Reply:
I have removed some unused test files. Because some commands use commands from other modules, I suppose the test scripts are not easy to add. 

Like when I run tests locally, exception thrown:
    [-] Upgrade 23.3s
      Expected no exception to be thrown, but an exception "Please install Az.FrontDoor module by entering "Install-Module -Name Az.FrontDoor"" was thrown from /src/Cdn/custom/Update-AzFrontDoorCdnProfileSku.ps1:144 char:13
          +             throw 'Please install Az.FrontDoor module by entering "In …
          +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
      41:         } | Should -Not -Throw
      at <ScriptBlock>, /src/Cdn/test/Update-AzFrontDoorCdnProfileSku.Tests.ps1: line 20

Reply:
if you want to use command from other modules, you should install them during the process of setting up environment.